v2/pkg/output/output.go

```diff
@@ -302,7 +304,7 @@ func (w *StandardWriter) Close() {
 
 // WriteFailure writes the failure event for template to file and/or screen.
 func (w *StandardWriter) WriteFailure(event InternalEvent) error {
-	if !w.matcherStatus {
+	if !w.matcherStatus && !w.matchStatusPerRequest {
 		return nil
```

Comment:
@RamanaReddy0M  is it possible to handle all logic related to matchers status per request in common/executer/executer.go . without updating  writer methods ??